lighthouse/types.py

```diff
@@ -12,6 +12,11 @@
 EventMessage = Dict[str, Union[str, Event]]
 
 
+class EndpointParamsException(Exception):
```

Comment:
I don't think this is the right module for exceptions and I think the convention (not that it is always followed here) is to name the exception ending with `Error`, for instance `ValueError`, `KeyError`, `TypeError` etc.

Reply:
This was a left over from my first attempt.  I should have done a better job of stripping out the fluff that created!

Reply:
Haha no worries.